appveyor.yml

```diff
@@ -17,27 +17,33 @@ services:
  
 before_build:
  - cmd: dotnet restore AllReadyApp\AllReady.Core\AllReady.Core.csproj
- - cmd: dotnet restore AllReadyApp\Web-App\AllReady\AllReady.csproj
+ - cmd: npm install gulp -g
+ - cmd: cd .\AllReadyApp\Web-App\AllReady\
+ - cmd: dotnet restore
+ - cmd: npm install
+ - cmd: gulp
+ - cmd: cd ..\..\..
  - cmd: dotnet restore AllReadyApp\Web-App\AllReady.UnitTest\AllReady.UnitTest.csproj
  - cmd: dotnet restore AllReadyApp\AllReady.NotificationsWebJob\AllReady.NotificationsWebJob.csproj
- - nuget restore AllReadyApp\AllReady.IntegrationTest\AllReady.ScenarioTest.fsproj -SolutionDirectory .\AllReadyApp
- - cmd: npm install gulp -g
+ - cnd: dotnet restore AllReadyApp\AllReady.IntegrationTest\AllReady.ScenarioTest.fsproj

```

Comment:
This **cnd** appears to be a typo. On the line 29 also

Reply:
Good eyes - I need a bigger monitor or larger fonts :).

I'm surprised everything built given this mistake.  I just followed the pattern that was already there.  It appears as if these lines aren't necessary.  Hmmmm…..